tests/storage/restore.go

```diff
@@ -116,9 +116,12 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 		vm.Spec.Running = &t
 		var gracePeriod int64 = 10
 		vm.Spec.Template.Spec.TerminationGracePeriodSeconds = &gracePeriod
-		vm, err := virtClient.VirtualMachine(vm.Namespace).Create(vm)
-		Expect(err).ToNot(HaveOccurred())
 		Eventually(func() bool {
+			// sometimes it takes a bit for permission to actually be applied so eventually
+			vm, err := virtClient.VirtualMachine(vm.Namespace).Create(vm)
+			if err != nil {
+				return false
```

Comment:
Should tolerate "already exists" here for the case when create succeeds and the the following get fails